protocols/plaintext/src/lib.rs

```diff
@@ -55,3 +70,130 @@ impl<C> OutboundUpgrade<C> for PlainTextConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct PlainText2Config {
+    pub peer_id: PeerId,
+    pub pubkey: identity::PublicKey,
```

Comment:
I guess that's our local public key? This fields needs some documentation explaining what it is.

Reply:
Yes, that is our local public key. Is it better to rename from `pubkey` to `local_pubkey` ?

Reply:
Yes please, rename to `local_public_key`.